pkg/kubelet/dockertools/docker.go

```diff
@@ -143,13 +144,14 @@ func filterHTTPError(err error, image string) error {
 	}
 }
 
+// TODO need to refactor this. PullImageOptions is different
```

Comment:
After converting to query, they are the same (`fromImage`). (See https://github.com/docker/engine-api/blob/master/client/image_pull.go#L19 and https://github.com/fsouza/go-dockerclient/blob/master/image.go#L273). So there is nothing changed here.